cmd/system-probe/modules/process_tracer.go

```diff
@@ -0,0 +1,88 @@
+package modules
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/utils"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/DataDog/datadog-agent/pkg/process/procutil"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// ErrProcessTracerUnsupported is an error type indicating that the process_tracer is not support in the running environment
+var ErrProcessTracerUnsupported = errors.New("process_tracer module unsupported")
+
+// ReturnZeroStats controls whether process_tracer returns stats that are all zeros.
+// Currently it's set to false to reduce the deserialization work process-agent has to do
+var ReturnZeroStats = false
+
+// ProcessTracer is a module that fetches process level data
+var ProcessTracer = api.Factory{
```

Comment:
Why is it not just called the `process` module?

Reply:
Lets rename it

Reply:
There's no particular reason, I can rename it to `Process`.

Reply:
The module name is updated.